diff --git a/dist/index.js b/dist/index.js
index 49e2dc1f44c81cbc19e7c3c1b179442280eba886..af85edf15faa26bc348e0e614169bb291f252e9b 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -4115,6 +4115,7 @@ function isCompositeTypography(computed) {
 }
 function computePreviewValue({ tokensSet, token, mode, logger }) {
 	const recursiveNoAliasTransform = (rToken) => {
+		if (!rToken || !rToken.mode) return "";
 		const computed = transformCSSValue(rToken, {
 			mode: mode && mode in rToken.mode ? mode : ".",
 			tokensSet,
